### ISSUE TITLE
Bug: Bot exits when connection to slack api fails due to network unreliability

### ISSUE THREAD

**Issue by david-igou** [Christian-Bull/slack-notify-go#2]
```
Logger: 19:12:18 Logging started
Logger: 19:12:18 https://hooks.slack.com/services/token
2020/09/25 19:12:48 Post https://hooks.slack.com/services/token: dial tcp 92.242.140.21:443: i/o timeout
```

There should probably be some sort of retry mechanic to compensate for when one request fails

**Comment by Christian-Bull** [comment]
Should be a simple addition to the postMsg() func

**Comment by Christian-Bull** [comment]
solved in the below commit - will close when pushed
dec118175f873d9677186fcb65cd04dc24479824

**Comment by Christian-Bull** [comment]
Fixed in ab2356c649720b35dbcdb5c09c539a4af4dc05f6
